src/client.ts

```diff
@@ -386,4 +386,10 @@ export class Client {
   private storeInGlobal(): void {
     global.__APPSIGNAL__ = this
   }
+
+  tracer(): void {
```

Comment:
With the previous behaviour (where `tracer` is undefined) the application crashes on the function call itself ("TypeError: x is not a function"), but with the changes in this PR, it might not error until it's used later (by calling `.createSpan()` on it, for example)

I think that the earlier the error is thrown, the better. Consider either explicitly throwing an error when this function is called, or turning it into a getter (or both?)

```suggestion
  get tracer(): void {
```

Reply:
You want to change it to a getter so it errors on `client.tracer` rather than the function call `client.tracer()`? Sure, that's fine. It breaks either way.